smartmirror.py

```diff
@@ -10,8 +10,7 @@
 import feedparser
 from PIL import Image, ImageTk
 
-ip = '<IP>'
-country_code = 'us'
+country_code = None  # set this here if you don't want news from your current location
 weather_api_token = '<TOKEN>'
```

Comment:
Interesting. Was there an error appearing somewhere by using `None` as the county code?

It should only work once it's fetched a geo location anyhow, default county code set or not.